app/mailers/mail_preview.rb

```diff
@@ -8,7 +8,7 @@ def application_created
   end
 
   def application_plan_change_requested
-    event = Applications::ApplicationPlanChangeRequestedEvent.create(ApplicationContract.last, User.last, Plan.last)
+    event = Applications::ApplicationPlanChangeRequestedEvent.create(Cinstance.last, User.last, Plan.last)
```

Comment:
why this change?

Reply:
That because I initially removed `ApplicationContract` completely, so I removed the only reference to it, here. Then I restored it in a way it's actually loaded from both autoloaders, and I never reverted the change in this line. See the discussion here: https://github.com/3scale/porta/pull/3339#discussion_r1238512693

Reply:
I think we can keep it like this. Maybe at some point we'll get rid of ApplicationContract alias...